frontend/fingertips-frontend/app/chart/page.tsx

```diff
@@ -1,15 +1,29 @@
 import { Chart } from '@/components/pages/chart';
-import { getApiConfiguration } from '@/lib/getApiConfiguration';
 import { connection } from 'next/server';
-import { WeatherForecastApi } from '@/generated-sources/api-client';
+import { IndicatorsApi } from '@/generated-sources/ft-api-client';
+import { getApiConfiguration } from '@/lib/getApiConfiguration';
 
-export default async function ChartPage() {
+export default async function ChartPage(
+  props: Readonly<{
+    searchParams?: Promise<{
+      indicator?: string;
+      areaCode?: string;
+    }>;
+  }>
+) {
   // We don't want to render this page statically
   await connection();
 
+  const searchParams = await props.searchParams;
+  const indicator = searchParams?.indicator ?? '';
+  const areaCode = searchParams?.areaCode ?? '';
+
   const config = getApiConfiguration();
-  const forecastApi = new WeatherForecastApi(config);
-  const data = await forecastApi.getWeatherForecast();
+  const indicatorApi = new IndicatorsApi(config);
+  const data = await indicatorApi.getHealthDataForAnIndicator({
+    indicatorId: Number(indicator),
```

Comment:
Another one to discuss when @gareth-allan-bjss and @steves-bjss are back. Why is the indicatorId of type number in the api? 

If we are not going to do any calculations with this then the general consensus is for ids to be strings.